debezium-connector-mariadb/src/main/java/io/debezium/connector/mariadb/MariaDbConnectorConfig.java

```diff
@@ -72,6 +72,12 @@ public enum SnapshotLockingMode implements EnumeratedValue {
          * set to schema_only or schema_only_recovery.
          */
         NONE("none"),
+
+        /**
+         * the connector holds the global read lock and set a transaction `SET TRANSACTION ISOLATION LEVEL REPEATABLE READ` to ensure
+         * that the snapshot process will have a view from the mvcc. This mode is invalid for rocksDB
+         */
+        SINGLE_TRANSACTION("single_transaction"),
```

Comment:
Is the name really correct? If this setting is not used then the snapshot is executed in multiple transactions?

Reply:
Also do we really need a new mode? Should not this be done by default?